betterlockscreen

```diff
@@ -140,6 +123,25 @@ failsafe() {
         --noinput-text="" --force-clock
 }
 
+store_dunst_paused_state="$(dunstctl is-paused)"
```

Comment:
Can you move this to the beginning? Does it make sense to make it also caps like "DEFAULT_DPMS"?